js/components/userFeedback/redux/constatnts.js

```diff
@@ -0,0 +1,12 @@
+const prefix = 'ISSUE_';
```

Comment:
Change this based on project path. It is important to have a system of naming, because all constants have to be unique
`USER_FEEDBACK_`